src/riak_kv_pb_timeseries.erl

```diff
@@ -127,6 +130,53 @@ process(#tsputreq{table = Bucket, columns = _Columns, rows = Rows}, State) ->
             {reply, missing_helper_module(Bucket, BucketProps), State}
     end;
 
+%% @ignore DELETE (as if)
+process(#tsdelreq{table = Table, key = CompoundKey_,
```

Comment:
This is just a stylistic note, but typically in other Riak code we don't use variable names with trailing underscores like `CompoundKey_`. I do agree that it's helpful to mark something that you're going to update, but typical Riak style would be to name it `CompoundKey0` instead.